File: C++ solutions/34. Find First and Last Position of Element in Sorted Array.cpp
class Solution {
public:
    vector<int> searchRange(vector<int>& nums, int target) {
        int first = -1, last = -1;

        for(int i=0;i<nums.size() ; i++){
            if(target > nums[i]) first = i;
            else if (target == nums[i]) last = i;
        }

        if(last == -1) return {-1, -1};

        return {first + 1, last};
    }
};

File: Python solutions/34. Find First and Last Position of Element in Sorted Array.py
class Solution:
    def searchRange(self, nums: List[int], target: int) -> List[int]:
        first = -1
        last = -1
        n = len(nums)
        for i in range(n):
            if target > nums[i]:
                first = i
            elif target == nums[i]:
                last = i

        if last != -1:
            return [first + 1, last]
        
        return [-1, -1]